t/full-stack.t

```diff
@@ -243,8 +243,8 @@ subtest 'cancel a scheduled job' => sub {
     $cancel_button[0]->click();
 };
 
-$driver->click_element_ok('All Tests',     'link_text');
-$driver->click_element_ok('core@noassets', 'link_text');
+$driver->click_element_ok('All Tests',     'link_text', 'All tests clicked to go to test 4');
```

Comment:
how about 'Clicked All Tests to go to test 4' ?

Reply:
Hm, sounds like the same to me

Reply:
There is a difference in capitalization in Jane's suggestion.

How about also using "Button" here to avoid the ambiguity?

Reply:
Ah, I see. Yes, changed to 'All Tests'. I would avoid "Button" because it's basically a text link, no button.